docs/refs/crd-upgrade-safety.md

```diff
@@ -0,0 +1,218 @@
+# Custom Resource Definition Upgrade Safety
+
+When you update a Custom Resource Definition (CRD), OLM runs a CRD Upgrade Safety preflight
+check to ensure backwards compatibility with previous versions of that CRD. The CRD update
+must pass the validation checks before the change is allowed to progress on a cluster.
+
+## Prohibited CRD Upgrade Changes
+
+The following changes to an existing CRD will be caught by the CRD Upgrade
+Safety preflight check and prevent the upgrade:
+- The scope changes from Cluster to Namespace or from Namespace to Cluster
+- An existing stored version of the CRD is removed
```

Comment:
I think you might need a line break between the lead-in sentence and the unordered list:
![image](https://github.com/user-attachments/assets/ab8c5ab5-b147-485e-80f9-7c66f1541f72)